src/Alterway/Bundle/RestProblemBundle/EventListener/ExceptionListener.php

```diff
@@ -5,18 +5,63 @@
 use Alterway\Bundle\RestProblemBundle\Problem\Exception;
 use Alterway\Bundle\RestProblemBundle\Response\ProblemResponse;
 use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+use Psr\Log\LoggerInterface;
 
 class ExceptionListener
 {
     private $debugMode;
 
+    /** @var LoggerInterface */
+    private $logger;
+
     public function __construct($debugMode)
     {
         $this->debugMode = $debugMode;
     }
 
+    public function setLogger(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
     public function onKernelException(GetResponseForExceptionEvent $event)
     {
-        $event->setResponse(new ProblemResponse(new Exception($event->getException(), $this->debugMode)));
+        $exception = $event->getException();
+
+        $this->logException(
+            $exception,
+            sprintf(
+                'Uncaught PHP Exception %s: "%s" at %s line %s',
+                get_class($exception),
+                $exception->getMessage(),
+                $exception->getFile(),
+                $exception->getLine()
+            )
+        );
+
+        $event->setResponse(new ProblemResponse(new Exception($exception, $this->debugMode)));
+    }
+
+    /**
+     * Logs an exception.
+     * Taken from https://github.com/symfony/symfony/blob/d97279e942bf3a135634ad90a32f1d5cd05a22ba/src/Symfony/Component/HttpKernel/EventListener/ExceptionListener.php
+     *
+     * @param \Exception $exception The \Exception instance
+     * @param string     $message   The error message to log
+     * @param bool       $original  False when the handling of the exception thrown another exception
+     */
+    protected function logException(\Exception $exception, $message, $original = true)
```

Comment:
True. I think I'll study a bit more the Exception lifecycle in Symfony, cause looking at the original handler, it looks like I should need it (as a subsequest listener can throw itself an exception, that would need to be logged here).